cmd/openshift/operator/kodata/tekton-pipeline/00-prereconcile/openshift-pipelines-scc.yaml

```diff
@@ -0,0 +1,40 @@
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: false
+allowHostPorts: false
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: false
+allowedCapabilities: null
+apiVersion: security.openshift.io/v1
```

Comment:
Can we reorder this file just a but ? 

```yaml
apiVersion: …
kind: …
metadata:
  …
…
```

Reply:
I think that's how the scc comes out from openshift,

Reply:
sure we can, 
i used what i got from
`oc get scc anyuid -o yaml | kc neat`

Reply:
done